queries.sql

```diff
@@ -10,3 +10,45 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
```

Comment:
- [ ] At the end of your query please add this line 👇🏽 

`SELECT species from animals;`

It is to verify that the change was undone as required 👇🏽 

> Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and **verify that the species columns went back to the state before the transaction.**